modal/image.py

```diff
@@ -601,7 +601,7 @@ async def _load(self: _Image, resolver: Resolver, existing_object_id: Optional[s
                 dockerfile_commands=dockerfile.commands,
                 context_files=context_file_pb2s,
                 secret_ids=[secret.object_id for secret in secrets],
-                gpu=bool(gpu_config.type),  # Note: as of 2023-01-27, server still uses this
+                gpu=bool(gpu_config.gpu_type),  # Note: as of 2023-01-27, server still uses this
```

Comment:
Super super minor preference for `gpu=(gpu_config.count > 0)` – this is what I've used in the backend as a boolean expression for "uses GPU"